<launch>
    <!-- Launch robot1 -->
    <group ns="robot1">
        <node pkg="multi_robot" type="pattern1.py" name="rosbot_gotopose" output="screen"/>
    </group>

    <!-- Launch robot2 -->
    <group ns="robot2">
        <node pkg="multi_robot" type="pattern1.py" name="rosbot_gotopose" output="screen"/>
    </group>
</launch>
